libcodechecker/analyze/analyzers/analyzer_clangsa.py

```diff
@@ -172,6 +172,38 @@ def construct_analyzer_cmd(self, res_handler):
             LOG.error(ex)
             return []
 
+    def get_analyzer_mentioned_files(self, output):
```

Comment:
I've updated the DocStrings of these functions to better convey what `output` means in this context (and also added a small optimisation for `In file included from` stripping)